### PR TITLE
Split the riskinputs in block of ruptures by the epsilon_sampling parameter

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1437195. This also fixes a bug in the ordering of the ruptures. The tests are running here: https://ci.openquake.org/job/zdevel_oq-risklib/421

### CODE REVIEW

File: openquake/risklib/tests/riskinput_test.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import mock
```
Comment (ptormene) [inline openquake/risklib/tests/riskinput_test.py]: Please remove if unused

File: openquake/commonlib/calculators/event_loss.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def event_loss(riskinputs, riskmodel, rlzs_assoc, monitor):
     return acc
 
 
-@base.calculators.add('event_loss')
+@base.calculators.add('event_loss', 'event_based_risk')
```
Comment (ptormene) [inline openquake/commonlib/calculators/event_loss.py]: You might add a comment to explain this

File: openquake/risklib/tests/riskinput_test.py (hunk 1 above)
Comment (micheles) [inline openquake/risklib/tests/riskinput_test.py]: This is actually used

**Comment by ptormene** [comment]
LGTM

File: openquake/commonlib/calculators/event_loss.py (hunk 2 above)
Comment (micheles) [inline openquake/commonlib/calculators/event_loss.py]: Added

**Pull request merged**